conjure-java-core/src/main/java/com/palantir/conjure/java/services/JerseyServiceGenerator.java

```diff
@@ -65,11 +67,18 @@
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 import javax.lang.model.element.Modifier;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
 import org.apache.commons.lang3.StringUtils;
 
 public final class JerseyServiceGenerator implements ServiceGenerator {
+
     private static final ClassName NOT_NULL = ClassName.get("javax.validation.constraints", "NotNull");
 
+    private static final ClassName BINARY_METHOD_TYPE = ClassName.get(InputStream.class);
```

Comment:
can we call this BINARY_ARGUMENT_TYPE instead, and likewise rename MethodTypeClassNameVisitor to ArgumentTypeClassNameVisitor?
Currently it seems a bit confusing since there's no such thing as a method type.